src/main/java/view/AssistedView.java

```diff
@@ -1,4 +1,153 @@
 package view;
 
-public class AssistedView {
+import interface_adapter.assisted_mode.AssistedModeViewModel;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.net.URL;
+
+import static interface_adapter.assisted_mode.AssistedModeColourConstants.TABLECOLOUR;
+
+public class AssistedView extends JPanel{
+
+    private final AssistedModeViewModel assistedModeViewModel;
+
+    public AssistedView(AssistedModeViewModel assistedModeViewModel) {
+        this.assistedModeViewModel = assistedModeViewModel;
+
+        setBackground(TABLECOLOUR);
+
+        // working with GridLayout(), so here are filler panels with the right background colour to just help with
+        // layout issues.
+        JPanel fillerTopPanel = new JPanel();
+        fillerTopPanel.setBackground(TABLECOLOUR);
+
+        setLayout(new GridLayout(1, 3));
+        Font font = new Font("Times New Roman", Font.BOLD, 15);
+
+        JPanel leftPanel = new JPanel(new GridLayout(3, 1));
+        leftPanel.setBackground(TABLECOLOUR);
+
+        leftPanel.add(fillerTopPanel);
```

Comment:
This is again very nitpicky but I'd block more code together just for readability